src/Content/Text/BBCode.php

```diff
@@ -1287,11 +1287,11 @@ public static function removeLinks(string $bbcode)
 	 * @return string
 	 * @throws \Friendica\Network\HTTPException\InternalServerErrorException
 	 */
-	public static function convertForUriId(int $uriid = 0, string $text = null, int $simple_html = self::INTERNAL)
+	public static function convertForUriId($uriid, string $text = null, int $simple_html = self::INTERNAL)
```

Comment:
You can keep the type hint and the possible null values:
```suggestion
	public static function convertForUriId(int $uriid = null, string $text = null, int $simple_html = self::INTERNAL)
```
But in this case it might be better to switch around the order of the parameters to have at least one mandatory parameter like this:
```suggestion
	public static function convertForUriId(string $text, int $uriid = null, int $simple_html = self::INTERNAL)
```
It might trigger more type check fatal errors when we wrongly pass `null` text to a function meant to convert text.

Reply:
I want to keep that order because this is the logical order for me. Also this can be changed, once that every `uri-id` is guaranteed to be filled.

Reply:
In natural language, I would say this method converts a `text` depending on the `uriid`, so this is why I suggested this order, but it's fine.